src/main/java/sonardenemeleri/sonar/App.java

```diff
@@ -19,7 +19,10 @@ public static void main( String[] args )
 BigDecimal bd1 = new BigDecimal(d); // Noncompliant; see comment above
 BigDecimal bd2 = new BigDecimal(1.1); // Noncompliant; same result
 
-
+BigDecimal bd3 = new BigDecimal(d); // Noncompliant; see comment above
+BigDecimal bd4 = new BigDecimal(1.1); // Noncompliant; same result
+BigDecimal bd5 = new BigDecimal(1.1); // Noncompliant; same result
+BigDecimal bd6 = new BigDecimal(1.1); // Noncompliant; same result
         System.out.println( "Hello World!" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)